Vaibhav Shukla

```diff
@@ -0,0 +1 @@
+
```

Comment:
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Index</title>
</head>
<body>
    <!Task 1>
    <h1>Vaibhav Shukla</h1>
    <br>
    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Unde ratione dolore voluptas. Cum neque quas vitae assumenda quibusdam natus atque corporis sunt eum ratione necessitatibus, ad veritatis, repudiandae, repellat quae. Consectetur dolores quo nisi iste iusto dolorem, ipsa labore deserunt porro voluptate perspiciatis beatae, recusandae earum quaerat sint pariatur reprehenderit!</p>
    <br>
    <img src="data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAAAQABAAD/2wCEAAkGBxMTEhUSExMWFhUXGRgaGBgYGR8eFxkaGBsfGx8eGxsZHyggGhslIBoaITEiJykrLi4uGx8zODMtNygvLisBCgoKDg0OGxAQGyslHyYtLS0vLzAvLS0vLy0tLS0tLS0tLi0tLS8tLS0tLy0tLS0tLS0tLS0tLy0tLS0tLS0tLf/AABEIAPIA0AMBIgACEQEDEQH/xAAbAAACAwEBAQAAAAAAAAAAAAADBAACBQEGB//EAEIQAAIBAwIEBAMGBQEHAwUBAAECEQADIRIxBCJBUQUTYXEygZEjQqGxwdFSYnLh8BQzQ1OCkrLxFZPSBjRzosIk/8QAGgEAAgMBAQAAAAAAAAAAAAAAAAECAwQFBv/EAC8RAAICAQMCBAUEAgMAAAAAAAABAhEDBCExEkETUXHwFGGBodEiMpHBseEjQvH/2gAMAwEAAhEDEQA/APqvE3LjeaEJ1idHPgnURGmQQBAz60Oy94lpDKcQrXWEz2YA9j07UwbyhmBRidRyFkZfTE+kyR0GaEeNtEZEBgu6x8TFN9tyPlmnew5Nt0/Pt7/0G4MOyy5ZTJiHJEdDRLVltIDOxaMkMQJ7gSYodjirZUNrEZE6jEqQp/EgfMUbUu2rvMN2ifYiR9aQ+p8CXl3Q2lbrtvOogAD3UDM46/nC3E+JFCyhmcoJaH2BzmSIH6Z65YuMxYJMAGAwI7jcCMkA/MH2rB4jwo27ZAZnT7xNsgjTkmDuOswZJ3FMgrSbVt+Q1Y8XvFQSy6pgL5kltRwMNywDHfEx0p7gfEGY6XLI2rrqgjOBnLCOhPTvWF5rM2oldSNpBiHZQD/Oqx1idx8qbslSzMy21Wfi0gHvzNqietOk9ymeon/2W3y2r0PShc/G2RI5j/nUVEstzanYieWCRAgYOcmZzjesnhfE0Tkc6gWhXCalzgKzL96Z39JM1e/4zbBhG1f0KT0nmzA9t/1jwaoy6lcXt7/g0rtoxh2nG7N396v5X8z/APUf3rNt+KMQZQxIggZiRkgkH6Tj1xWmjSAQQQRIMbg/Ogh3tP7lFstqMu2nECTI7yZz06VLtswYd5gxzGiQe4+n96rcBg5Gx6f3oHJt9yC0erv/ANRqvlNqnW2mNpMzO8ztHSKJnuPp/epB7/QfvNAKTRSwussA7jSYM6t/SSJHqMUYcGf+I31P71XhmCltTjMQTA74pkX1JgMs5xInG/0kfWm0uwlJvfdAP9Gf+I31P71z/RH/AIjfVv8A5UZeLtnZ1O33h1iPrI+oqy3lOzA5jBG4Ex7xmKQ7ZneIjybT3DcY6VJALEaiBhZk5JgUXhbWtFdbpIInDE/jPyoHjwD20KhHGrUJZQCNJkqTgmJMztNW8FHl2zr0pqcmNQPxCRJ7x6md6hb6q7Gjpj4PVf6r8+3oNHg2/wCI3+fOoODb/iN/nzo13iEX4nUbbkDcwN/XFdN9P4lzBGRsdvrB+lTM9sB/pD/xG+p/eh3UZGTnYyRuT3A7+tMrxSHZ1Owww3OR9RQuNOU/qH/cKQ03YvYZtTggadRgzknUZERiMZnrRA3xT3/QVyz97+pvzriZ9pPzIx9MflTsJbsWPD3CxJ0FCcoVB7dTHb1qcVaQiXRAYIYsNlYyc9VmCfYTG4u/Dp/G2ehdiJ9iT16CkfEuEUnzD5nQwuiVIHQsAcQZhpE7QaCPIS5dCOALYJC9wDJwNRnos7A7jaucZ4naS2Tc0szEqUHMBuYaAdKgAySOlI8Vw14KRb5iVly5kkkGFAMEH4RKsPUUnZsXdrhctqXkOtcMGEi5qOkybm2cb0myzHFN7geE4m2VlQBvOkKQA2QpLLsNQgZJotq4qHD6Flcn+bHxYXERHelOPtEWyV1s6mY5mOJJUsJDYEdCDtkib8LY0H4raFgZB+JjMbGMb7HvG9VxyPuSz6SMk5Ln/Jqm5sxJ0kg9STECcEgLGYiOtWHCBtRITmAMRjP8w36/Wl7nDMwwrATtyd91xnc/Eaol0gNq50bchRpXBwROVxmB+tX8nNUa4YS9w6jSAFLsIUfPEamMDEzMYr0gt/Z6UJXl0qeq4gYPUfpS3B8NoGs6tTBJmJGZjlwck06gx/nXNRZrjFRjSM9fD7gP/wBw5gkiQM5JAaNwMbRP0irN5EF3uXCwCwBMkHeJ3OoD2BOwMadI8RYvknRdVRIiUBxnHT077biYoGBteEsFg8ReOInVnKwfn1B6HNMcH4foIPm3HgRztPWc43zv6DtReARwgFxgzdwIGcgR6bfKj0ACvcGLvK3wggkfxCGBGDsZodvwwWmW4LkaRpyJ5ZTG+/JBb19BVxd8vUZLSRuTiZwNKkxirL4iSYCztPx9RP8Aw/8APSgbFLHhtsKoFxeVeHWYH+5eR1+8QB8hXf8A08L5IViwtfDGnooTmlhJKgrMdTtin7R15a2vuZnfbmUGIootL/APp/agR5/iPD1ucMtly2lAIb7OdKpoj4yIK6gZ3DMMdC3ODBLO+oq5XUsoAYBUDFzGoMFPfSO9aPEeHBjIYoIiFW2R781smTPeMUSxwmmdTG5P8SpiP6UG/wClACF/w9LjMzFk1OpgsJJHlk6SjSMWV+rH2N4hwNty2pwoZbYgbr5TFgymcEM4IMYIBp9eHT+BR8h+1Ca4VJi2Pf8AwUDSFLPhI1C4tyYMrjHwqhmDn/Zr+PfEs8GbSohbVzyDEHmcMZ75J+orRa72H1kfpS/FmdB/mH/cO9DBci9u4NTLMczHr/EevSi2nEbjr+dDUNzRA52zP8xnEVFtgAGSIG8n54ODQRbabsVNkFzFpDmQ0DBkjIxAxODJzVmsIHZtEREmMuSAAs/e9u8Ua1AmUhcnVgznt8Un2zS9y0GuqMiNRjbAxOMEyetA23VBeHDhgGVd8kOSSYPQoO/fGBS/i3Atc3DsDsoZQo23kAwY6EkdK5eYW3JLO4iILqImDOdPoN+lBtXQjfZ6WJnkVgGUDMHVejbqKAtpqjA8T8OKwEDOGUlRA8wKZjCjmgMwgZjbO9fA7iW+YajqADRIgqSBJJCrOrIx0+Z+NvEkBbZtgnVJKo063gFipBICiAGH0pWyoKkWwRLTqESdOiJKhY+9k433NZsjS2RozTk8fvg2eJ8QUCIPNJYAqdM9J1Rq/lBO315xUBVVbYdCGnmxpBAJBJ/Ce3rSXAXdWktsn3yVlWGeaTJyIx6Z3ht+K50uYnUsuiEyuoAguFIAzMEiN+lEMkk6ZzscV1pPj33HfAi2lpWE1oUOwIwIWGMgRM9a0rvidpX8tnAf+HMn2EZqyhWWVOCRkRBz6Crb4JMjfb67VpNTdg+I8StISrONQ3G5EicgbYoI8csTHmZ/pY/jFMx6n8P2ruf4j+H7VKiXSDXj0FpbnMVOBCmZ22iem9ThuPV2gLc65KELj1NGsrgZOw/zFX0e/wBTUSADjvhP/L/F/N/BmhWCymQpMwPhuyRH8wOnYZ/GrcWAiN1lk+KDuT/FSP8A6hdAPOJAn4Rtmfu79qBK63NhOJYkAWzmTnWPxKCj2yeqx8yf0rCbj7v8QjY4XDR7d/z9q6/HXcjVtOyiTOwErttnO4oGb8f5moR/mawrfiLg8xwTtChgO+xBH+TTA8RXGprq+4T64FAGutZ9+63madJgkCR5vX+ldI69fpVkv2ycXiPfSPTqvpRrQDTpvFu8FTH0FA06OcSulZAY9+Z/wChjQGuahbMEc4kHV/EOrgE/SnfJP8bfh+1LcUseWJJ5hk7/ABCkNCNtpuXRJhSSQVxktsZ3x2oqKwnUFAxpElvwwBHt60n4jwpLajBVXJMxiHG2JOCx36VoIWAnRzEwRIws7z1AGY9aY5PfYAnH25lr1vaRzACC0dWz0E9+1KP4houRhi5IBA2BZoxMmBBMdpxWvccicTAkZ3Ocemw+tZPFW/NMeUrPAILAFUbeGJEEZggGYGKCt7ugjvd08mkPvlsM2kCPhLAHvpB2xSvG8a9pWa6ly4FjCIdJPcjvOILRtgUe1wpAcvZtuFkKq2lDOBnllyAJkQY29azuOtLdVhbIHMoRRGmFK5iMAkEyuYjtSbotxx6mZfFcRquFtQknSdOpSuSBpIaGEScjp0JikbqkFLgKNnUCSEOonTIKnM6iT6Ce9bN7wxyjC5Fy6oJR4A2U4XUSJJGeUepHLVeH4cTLryjUhOpQwcqDDkaQTOBOZM7Vjld7mjrS2oQXijbS3qIduZRbZMcuBLEZPw80fMivQLxicq6lKEAZtkkjAMsGge8Ymk7d8c2lrkoJGga9GAxDYPqI6ekCApwaB21K5bMsQeqLGlSVIMSc5gGPSWNVuUKULdmz/r0LEWrh1Fhy6SbZOOunE4mD695eF/Vj4XAypkEe04YeokVj8KGK6QQLaFTILazMyMN1OqJYxAx2Fw9wlVe+F0sAecXMA5hTdUKJGSNR65OK1QYZF3NxL56qfyx3g5oyPP8Am3vSVu2p+0Ba2uIChRMD0Ukj2PSi8SjFBD6CdrkD3yrCBPr1qyyCkO2PhFXrGs8NxBB//wBB+8uEUQTOSDBkSpGem2abaw+h18xwWbUGgSokHTk5GD8jUBDd20GBByDHfp7EUunBrq+AQIIy25npqpM8BcgA8RdBwZ7x2gmBG/rHsWBw0h1NxzqMyDzLzEwN4Xp7Y3yQi4t0FbgFiNIjOJeM/wDNQ04YQupRrgA5cqDuck5zPqaBxPBIVAJcBRpkvE75J3mTJ2nE7CpwNqxblLR5mGB5gY4kiAWO0k4oJWlyhq5wiiOUEE5ktiBvlv8AM965/pE3CAx/VHtBau2eHIXmZmyI1wSBMbqKaJoIqudxO1ZCsxgKOhBcE9TOe+wk01buOBAg+8z9SST70jxniKyEVxncrzR22mCYJkiAAZpNuLuHkaTESApztgsAAZkYUEmI7gDaLXBv5e/f/hsNxV0qCoXUR1nT06ifX8K5xNxyyYBGoSdoyIxmZ/CKxPCvEFRirOioQGydKjV8ITVGPi6DYHc52f8AUK+kqwYFlypkfFG4x0P0ovYOnoaXPz8yrZ1qVlSxnbMtERQDxNxeUkSCAAVJdpEg8p6iZMRKt8pxPl6zqt6mDMQxSYgk4aMGpf8ADUfnuyWAOQSAo3gAHPXPXPtQV0036k4u+Tn4P5nUyfQLv3M+mxE0ewWEDkjMnUZ98jPtO0UFboGl3xKwoAJCrjEgbnEn0xtVIsdlMdWBIEdNTYAG29LqLlhXBxr6MC7mbf3RuDJ+LSszMiJ2AnE0K3xlpyul0hCZJYAkgFYAPvv6R7TztZCDXoIBkKRqnfP8InJHcfO73RbOkISCZUKpMTuDEkd/njAMQZcuKFeO8TAhEORmSGKhRHMSpwIPzzWG/FKVJuPc5SUYaCNSkk6pR1kfDp1bSQJmRtX0LIzqVDTIBUZAwQVkEDJ6zOesUG9w9u6ruVQuU5tQUhTzLpI1EDYzBMzuQBUJR6kEot8GK/FcRlQFRHXACABwo5pkaDK9j0jrnUwoDnXEBittVCENABhxA+9tuDFUvMgOhAfiwQHOYCkCVggBZA15gURNAYHW2lZllcqCCTEK04BMTI2A6RUEunYy5U+q6KpwLXQ2i7dHwypaE3kaUA5huN4kRkCnl428FBcnSYhxyKBtLQupBPpAG5xJX89Z0JeuymnRyglwNJKg6JaQNzMnPSaK/CAtrCW2DKTzIdZKkja4Ro3USfoAKshKit5H3CWXUXCRd03NOk6riupAJI+8GMS0THxH0g3DcVbDuW8sODAbAa5qGSF3ieXczB7VjreZSV+yAPwIW1Aqe4nSSZAIEGRPUyxbe8CPMtFH/hQFkgdYQxkxiN98AE3qcWWbG5wvFKys6tEMVaQfimMg5B2pe5aFwhzcvASQQrFbe2JHTpnuaHw9u2Vl7ARWYnMYZiZ1gEEMT7+/ZXw+3ae3DcSTucaUXmbUMMN4gb9OlK0+BNVua13hrTsSXyQOUNAgZBgdc7+3pSt3wzhVguBEHL3Ce38TZJrnC3iFAlSWCsAzSZIzGhYgNImTgVm+OcWC1pQhA+KdLqTjTEso6nvnFARXUzbK2WthAoZGM6SpMkGchszInO29Ft2lUgraUf0rBG/oKwPDeIRrbINYYPEByrL5h+LVPNuMA/d96bChGVbhVZyBduBjE6SJdmJ5YaRAJAG5oHKLTo0+K49EjWdE/DqIAMZ6nf0rP8S8SLhltxEc2o6fdZPpvAJzGKynvWCCS9t1AuAiyVJiTE+Wg5Iy0n4gN8UsniPLpjlAUk+UQivAleWVJz2BHMDM1GTotxY7Vv8A0Hu8Y2A6DUS0F2MJAwSGADbgwIG/QzSHAcU4UtqBWZAMQ8RpACmX+Q0n1ofHX9ZBUggxBCjSI77gneAJJkHoJNw9t2uqp0l/uguAQwgrrbQCYkCB2jNR9SxRXnt78xrhbbMfjS3KvKwNZIbA0kSV2wRB016HwzhgES4DJdkML8A2EAegAHuDtsJ4J4d5CkGGc5ciJJgDEKuOUwDt3p4IoC6QAC6mAIGSKmlRTOUZOvmL3LZLPgRzQQTJ+IMNOw3EH9qBb4p/MUM6xE/CcjIJnzMjaGI70XiSS+jS2gs+oqc5YYwQQP8APe6eF21MxqxHMAfnMTPzqRVN7gLl+1ZuBSwUMCRLYYs3Qbcv/wDYrt64gZ9VwguIgGYgESAJg53/AKe1OXOCQoUCqoIIwBj1pNkRwGKAGIMbqeokQfT5VBo0wlcRRuNQIGRwxVcKP4wBvCyJiDthm70Cyqtpd2dSpMmVOsgEf7smSJzge1M3whc3GW2QBCE5ZiN4BA05UjczHtK9jhvLdZQksCSuudIEH7zemT1JPaok7OLw7OxkKqowKwvMMB40gRO2xkT12oPHeWrPpuaJgqmRkBhJR8MJAOw+E+9HbiWyBbA5iDDjUOYwDyxIAxDE4mq23JLB5ZkECFIkxupE9DA2+M4ETT5IdSgnSKW7ztJDK5U/7plKwekOBERODkjfEUHhbiunljSIJFw64YMDGoBAcxmdQjuYwa6j6AFLLPLFzUx3AP38GJOO3SlOLa4LoC25wAdLYgHGDCkkAjIO3oKmorkw5JTm6exbiGZbegILmrC3EJ8xX6nMk6ZH1PpVLtjWZu2Qr6tIW2uGBTVJBOqcbRsIGTh3ieMthmTCcwUhrhV2GOY6hzdpnrU464G+0Vm1xjyiZAUzBklWGNtJOcDrVHT8iyM2lS7ii3WHJcEBSxW1pDITt0UETMAnGR1xRbXCgWxd8x0c9wutgSBmRqUAtk46bQDSpZx9obRcWwVBbQls6jJ1Ky6gTAgkDND4DjH0mQ4TTBcKI1gz8SCYycQSfSgaVjd/gm+FyCxMAOmsgzhgznJgk9OvrRLvmLMM2pQRCKGWGx9kCwwTmObTgTiuWeD1owUpcBuQ0LuSJBIJgxrXmneZzspb4O5qcoDyyBqUArpONPfM7Hq496nLfZmjGmlTo1bfFXrQUFWbVGSHdpXJEOV0g94+u9KeK3r11VTy7aPf1KQSVblxztBldjgT7dIPObnW2nNpE3EWAFzDwxK5MDJmKz+OQMArqGl5QLodJJUHSgA/iBiM6u9W43Nur2BbO0txjjuH0A6nS8QGABf7aYnAJM7FSOkA5g1kMr3CEVTAYghNJAiSQvIIyMTJxt1r0fB+FtaDM7QXSeVR5qkAYnT0yAYJkkyaW4rw4H7Rr7CSwBNhixJAG4IIiASCKundbEXmndJrv/Wx5u5F3VpcgDK+bdksNI3UDS2fX8qe0NygC2pxoCDCKrRKsACGPxSTHtWufCrYBfzCWjmOAukJqhQWJHJn/mB61kcQBrFpASJUM7ggy04XQo1NA7dsztW1O9x4JNv3/Yzb4h9erQqm3JlVSTqjL4ePckb5rT8Fs3nveajwiFtiWRyegJbuT2IxtMVm2tQUIiOVBAAZyVIImW0hZMADmP0yR6D/AOmILXGBSJKgLmdBgthiFziP5RU1j36mW5nUdjX8tgWbU7HTheUCRO2B36mlLV1nuITZgKQA4eY5hIiBIxE52pzjmgDMTyyNwCRMR1iT6b9K4lxCECYGpYEacAjYEDFS37GSL6aMzxLxVrVwqPLyxnU0aQSRmM5MAdKXbiOJB/2hzkfAVEnABKyQJjvitlrYY3FOQSwP1P71k3uHWQuvUhJZT5nNPYcpkCR1j9Y5YzpOJiedxnKN92LHxPiCxmQQ0Ro+zODOBzMOs6owKC6/ZOEJEZ8wxpBHUtEbn8O9MPwzqrEsCQVOhWkaTiWLQxOWMyBgQKXu8YsBVay0gAprmBIEkAxI7ZrLNyumT8ZyR2xbK2w1y4rHZSdOlgIyJPQ9Bj02hm4qXbhcoWUKo0wrR6CcCQTMdqRu3B/s2XSoJ0hlYkl5BICoREkxiDI9q5wHEqraGuJpUmVCabhO3IwVTuZx+tPHPszTiyyX7mOcRbCuIV1SNQRVXliJ3GFzOD1PyFb8TttDG8VkFmHLIAJAB1z6z89qG/2k/aBkEgQTrAJkaiWJjlj1k70uELsV8xUfUQSu7atwFaTtOJBE4itiW1FXifrckP8ACKl37QMWEwpDEDGC2DAJ9OnbVQDfT7RS4VGIA8xtQcrzDSXbaZn9ZqwFxQsIy828iQdOkYB0hY7kiehOaH5HJqv6nMrAiQNGVgKABn64FN77EEt+rt77DdtrZBVGRWJyVYSIP5fKhW2DMQtw3CAQWJEAkiIIjOIx/aguttlLW7bEyckEZZgTBYiYjpMRAii2+HZTrUBdyytJLEjLSjQCe0HMmKkiD2uhpOIddlaQdw4LEdT2Pz/SuNxJA8xvNZXxpCKQCG+9CzgRuPQ4FKpxd9tWhUdpBINsgddiX9MexxRla5p1G2+vHw3CqRGIAY59CBNZcs62ocXJMX4O1qa462nU50g4MEQrRpGDpAwxgt2qiXgjpItzjlZZeSZEMzkg8xM+u1OFvMUhgqgRi5ZZyMAAAnTEQOnzod/hlIYh7iqqmILlWgQcfAVwBpjvvSSXNHTxtSigI4xGuKGVFG2kWpIJC4wZnVy7bjEgiu+IPrcc581V1oWBAkwOVZgSBB68xB3yGxY85QNFsqI0hgoZCu8ow6/LEHtAuO4AoytoflIX/aDyyT8QAK8hIiNgYFaoY0tyVUN8NacgOwLy06Qs2yNQuEkBS5h2me7YiKbvCVEWWZdiihwwIkSQSIPNgnMHtWV4C41kYGuTqILorLDDScBiZJJPYDpFei4eyAC2m4pHxPpXbqFBGROx09/ak1uZcq6Xa9+/8nnvHPEpRLVjywCCTFwl0Ucgl7bDTviScCDtnH8M4YDmRGbOoXADzFSVMTqVgJUZIxiMRXo/FbCuodRfnmIcAgDY8ysRpX26g4xTVq5beytuzdZgmNTKcggKcmMgnvIyPavJB0y3BqMbglFb9zP8OKG6bd5nXCwGCwZESVdYI5DtuTXovD1CRbQIYjUQ+lpOSSirvnqax/CIVLpHnKynn1lQvLtgBuhDfCd62rXiKBE1auaPiWJOgPmQo2IEwMkCAZgjdKxZJ3sMOstIExAGo4mCT3jdcxVvLMqSFHMsRvv3rzh8aIa2x5ReKk6iAACQAY1amjIJ9BvWtwnH6zbHKrFhqE4MMByD5qTnZhvuJb9iChb+4yto67hLGGYwNoyZIIzn36Vg8Rw9wHSx1JLSoVwSxGOZtwTudX6R6Mbt/U351y6wA5tjj61bKPXCjlZZ1lk2u557juGskBQNBIUF1GkhBnJjrBwRG/aaBxN60zqjFSFKsxYkMWBGiIUBtj8Pr873rK2rpZLaFScstzS3NEagu4BAyekExE1q2LBPNqAU6eRCGGP5onYARWBaed1Ra8qS5Mzj7MOmjWrG7IBAKkkEMQQZjRqMTHoDkctsbnLb0MEDSFfIk4JDCBkNuTmmfEbhN22Uf4Q8wmqDjoSIEEjHf1oFpbrEr5gNwZ5LbLI9y0Ej3ozVCexLSwc6v5lbjynOjmMwVjVPcHlePelVvKvLoIVm+EoCWH3lg47n5gVpa2FuSuptoXlJz2GrMSZUmd9tlLIXUSzPbPQMzmCQM83sI2q2Oog+TV8Pkh8wD2LejWwtpg6CsAzuMxJIjpHqDvWgXRFBU82Jn4zEapJ+9HejvxSjSI2JHIywTJU4JztPpND4m6uqVeHgjZjq6j4eWfYzmMTS+JjW6Y1pZJrj8lFYkSI2EuMZJMlVIM7daALtpYYkuJwzpkYnk5B22zvRvLvHDKASRJSAJMRhnmB1xJgUv4o0Ao4DnYMywRHMYgmREAwe1HxCcaRXlwShu6K3rshbjWQ/KcnSQwxzAMBpXfpnGDvQuA8nQSbJYEsTqVZXeJLAGCOsdTnegHgGCw6swJkW5fSsAcpOhpI3yBM+5o3CWvLCvatibgmQdXw5ghtITHr3nNUc8jjjcnUa+gexxAuBkFpbgBBGE0IsbCRuMg4yI2mjGw/KGdgOYBECBFjI3Bzg5xjbNZ9kSxOtQzHQVACsNMjlMN2frGN96LeKgENbuwCAGkwGJABmY7fQ+tWJtLY3qtti/EcICeS3cVlLS6qqsQZ1EMevXpuepqWvC+QvbaOYHmxqWRuBggwY3mavbv48sKxWWV8Nzf8AOAY7HtWzwtpAOUn4dvMZ4/6ia0aZt3fBm1Wo8JUluzG4Tw64mglsK0yAuxJGxWYAPWfwmty0qOdQYk7x5jECduUNHyinCtVsqBMADJrU0jlZNTkyJW+DlxTGACfUx1noDFJnhE+01iCwnDMYJBkiYz6wOlaNBuIdQI3EfTMj8fyqMlaI6fJ4c7Zy01u1yqHMknlVnzMZYAxG2TMCjX7mB9mzTOABI/6iIq3DxmABkzAjPr+fzrt68FiZzgQCfyFUHUTvdGHxPhxZ21nlPMAQAdy0SuA2J+fWDTnAcLoW2dFtZKfCDOWB36D071R7tm4wVjcJmCsXAhO8mQAR+GBVuBYwis7EqyiCoCwGEHaZ0kdaRNtKvUaCczmT8TCOm/ar6T3P4ftXE3f+tvzq9aY8I42ZVkl6sFctTvpMbSJ/Wg3ODBx0O4BYTg9mpuluO8yB5enVP3piIPaghHnk8/4xwYVyULWyqiNOrSdTGZIUwcbn+9N2f9QbeWHKuABlwM80jE4EY3z6KNaN2WctDEB0lgkLqYn7T0Owxj0pu2LJBNogQThdJIKqFJzMcoC+wxXJz/vZ6DTK8af3+py3cdW1C0jayCWNzOdvu9ogYGKjcQDrZVAIK5uLA29YP3sVLfhui3JOohUiJT4NsiZA6Azt867e8OgAowH3IILbtI3aIn0J7dqr2L06W5ODUMztCajjWACCDBOQZP12ApggnkYgsDqB0hcjIETncbZiRSTcK1t/MDEgljAGZJ33g/dgH5bkUG7xVpn1FZMgajjIPae/5Vdiw+JvYO+45f40PhFaVIBIXUAcHoTOCD8xRUtmWLksQZEqqgQME/Qd9sUurgFokEXEGZweWfzyeu1F4m8r5a2XUSZNsMkenWRtt36TSnjcGkZtXtChP/05gcIgAkDRKNBG2pSNXXf1Peg8bctLurb8wZSxiATDGQx2HXGO0CujW4U2BE4lNEZMZtsxO+cU2GcKURHB+8zNqAESQksT1AnTA9SIpxTIYJRjFtlrnDtqDB9AEaVKjbOJPMD1zB3oXE8Iztp+yZo+IIumOmrVqJJzAEdaPxXmA6SovagAodlGo52C28deu01qeD8L5duCgVuoBkbdOw9KuxYnJ78Ec+sjGH6KbMPg/wD6dJ5tRXABEHMD1J/DFbfDcKbaadWrDZIz/enwKqMn5VthBR4OZkzzyO5f4RWWncfT+9S2D3+g/ea6gwDnYe3/AJrtpdz3zU+xVex3T/Mfw/ahXrCtytJBBkSe47UZVjArjbj6fr+lIiILbW2SoUwNlVCEznc8s+s96fsHYac9wBt6/hVL9nVEkwOkAz/1A/h3qnGWAVmASDI1CQPkCOlRkrRbgl05E/PZjIwxn3H+fL8fWkrN8F9I1QWQ5VgJVgpywHTR+NS54fgs2lyMrKkj1B1M2DgYj8qIzhmtEBviBBxABKmG9cj/AKao5OxBb178y1u0A9xupYg5PQmMbDc0aqIeZ/6m/Or1ojwjj523klfmSqncex/SrVU7j2P6UyoW8TaE1div5gfkSPnSYtC67C5bWQMagrTsAR2jP1MdaY47h20klywkEKQsDmnEAHAx3pTinuhgyoCIbSwbpgwykbexn22rmaqP/J6na0Lbxejf9M5xP2QAFzSCTvBMdY1YmSP77gSIXQ6WLAyI0sSRPo6/oBIplXuC2zuAWILYwIHMqxJzHqcntFCW1CH7JoKsGIcEtJg7NHU71GOJv9xsiklSFuLu+WAGVxJJDiWmABkEEycd8T2qmpWDMXYKwVpJKxMg6ojoBg7122uJgW2UZAUL7YUTJE4k5261nWkDcxQBjAAPmLBExEpvsZOd+0VugqjSL4Vyx5WZQuhfsyRBuMVM7DGmY7TBxttLhTIV9MNmMYaOhMfX3rMsLcDOrFWyRgmSpLAbrg4E9PkBI28SILA3mJEq3wED4xqiATDASMe9E6qmZdRlxOLj1GqeBRbZm2SDODuJxsT0HeKoq5C2gyk9CJVBJYH+Ukj0Bz70AcWed7ZUDSvO6ktkk4gxAnG8TnbOv4Hq0kkW4McyMW1HrM9vc1CGOzjTyuilrw19epnBIAEguDEnA58CtNFgR6dSSfqcmu9flUJitSVcGe2yMcVxRj13qoM5+gqDcyZn8IqQzifCPlRFUCg2Iznqdz6/lRtAiOlJkZFqq5yPf8wRXSox6bVW4Bv6jPzpCL1w1zzB3H1ql15EAwe4E/8AmgBJVs3DjiGbAmGUrA74j07n3qcHx1s3PKtmRqVtjy8wBGRtOR7ntV7yuZ5pBI1B0BUgegK5wMkmiIedRpiSpwANm7CY371S4OjpR1UOpJb20g4tgsxgTqb86t5Y/wDGPyoaXAXdQwkFiV6iWMH0GDRROZj0/vVseEYs6rLLbuzmj3+pqunO52/P39qtJgYE4nt61wfF8h+ZplVmZ4tZtjLm4NW5VoGB1HeJ+lZtnxYOioGUOp0nUY5lwQpA9CfoMzXoeJsLcUq6yJ2PWOoj6VmN4doY+Uqr5wAOqcEAkjlMyR1nBX1EZ8+Lqal5G/RZ+mah5lL/AIkHtxq8uSskKxgT01oDOD0x9KNwShkZRcVgpMhcEAzGFOMbeq0u3h6o/mNbYmCWuBicgg7E6iMHEYmMigW+BslWe2H2BWAyjcQeaFJgD/8AY9TVZ1m6fAS5bJ8zIiAJaZIIBEmeYR094rgvlviAVscsgxmCJVyQOhYiBNDPDBGWXuqMj7XQylgZA6n7x2IOB2plL1vXpcKSfYjbpJ5R6QPxNT6pVRy5anI9urb+BN7mtSHdUUjClZaGIydQO23rj0k/BeB6ckbmQTlgMEZ6ZExnfvTHG8Tb2JGrUm8TBZRufnWqeJTHMv1FSxwUv3GaWSS2WwnZ4HS0+bIxjSsmO5jNOKYG6/T+9Fiqr8q0bFOwPU04jbt/euXkOkknoemPzoi7/X8xUvnlPsaAvfYNxHmEkA2tJ7zMdRj5/hSNvwySJFsbTD3Nh/zCaesIdRjSFg5G/vEe/wBKpwtlSTqQPJkOVSDHqMnr0rOdsZtKRA1pAj7pn6ljSdtBA32HU/vWjYCBYTTAnCxAPypFNh7VLGYNZ2+pzyx2+ufzqtxBBMDGdu1EqtwYPtVphLUO66iNRAnaT+VEBoDcUobRDT6KY95iAKADaf8AJobiGT+pR+I/ai0G7q128DTqBJnIMiMRnc9elKW6Zbp1eWPqXQZY/wAzfmasfShrdGpxOdTH8TV/MHcfWiPCFmd5JV5shnG3rXFPMcds/wCe9dd46E+1ctNMn1/Qd6ZWXpPxZfsmIMFRqBO0jvPQiR7GnKzePt3JP2jaG+6LQYARBEjOcmk+CUHUkxC1x+rm8yVHTQ4OoQYBJii27SaPNLsJ+LmIHMdoGJz9ZoNu+sG3du6dJAaeVu4yIKkxORtHeiNYthUIJMgQ0h3YkYyQxYADHpWW7PSQcVunt6e2D8xdSwVZWVhIV2kxP3RnE5wTqxtR7PH8vOGxMyhBMCJ5gO+dhv0yc1LxLlwt0LOx0iTIYnMQcDad8npXDcUkwACdmKs0jVGYB5jn0xjpUJZEuTLqNPj4XIw+XLmQtvSWDQdQLEA4nVGfWUXJyK3ktgjCj6R+leYucAxRCLgcnSunRkgdpIIj4s/w+1bXg3CaFCaXXSBmQFMAbgMZJ3mKs0+Tqukc/PjUUrZpaj2rgJ7VGNSfSthmOEGR7H9KlxRpPsa6+4Hof0ql1MH50gQyLmWiPg/eueGElB/DAjaPXYn8YqtsfFj7p6epofgqYYxjGY3jG+x/Os52yeGPlgD/ABGM4z1xH41EfA9qN4eBpaB1afr26fhNBAED2mrMZh1lfp+v9BalV8wdx9anmDv9P7VM54GzclSD0IHTYgHoT0PWPaqji7YHxA9OXOR7e9VF2GjuMCDupg7naCNh0yciiHiBGTB23X26n2oJVfCBOUZom6CewuBfrGkbf5NM6ACsfxr+YqlpHEy2rsDAj6DJ/wAiutq1LtGpfedQqMuGTxOskfVF03b+tvzNXoNq4NTr1DMTg9WPXY7UanHhBqE1llfmBvWEOSv0GfwzQ7QkmI0+xnt3ztTJFKcPEfZsDvBJLbN1aZO5plZbhW1A5BjqpIH0kx9aMgBE5+v7GhKrg6mYR26fjRbbZIn1Geh+dCCWzPP8UHV4IkErJdwT0AbQijmnEZBEbRFRrTQAVCs331UHcfemO4yD0jtO3xXBI4aZ5gQYZh0joa81b4Qs5UswjckbE8uGDSpyT/asOouD6o9zfpczW1jFw24BuiXWdP2eDy9C0g+89yYAwrgW8BAZjJQgyOvLjpgZ9e5uJ4KyuA5R8AMSTAIiFMb52mZFAv27jKAxATYG4TbPUid4XlEBhPsN8jm5O2aknz7/AAU8MW414MjIFIJULp0vDATKqT2OPSvT2LVyNTv8gAR9SAa81wvDszA2CI2fyyC2YzqLKI5REV6jgbZCQQ/X42Bb6gkRXS01dBj1ORt9vsE0n+I/QVFU/wAR/CifKuAelaTJYFk2MnqO35e1S6mDk7dz+9FboI/yDXGXlM9jRY7D2beDE/B396D4LYIBOrHVfkIO9GsbH/8AH+9d8HswgMzPoO/eJJ+dZDsWd8OsgIc7lj29PntS9q2ABgVXgbjDlEFTqJMbEDv+9GFW4+5h1j/b9f6JVL1oMIM/JiP+0j6VepVhhM+9aRQWESDJBMyJzuCYjOI2GQKvYNrUNJt6ugUyc9h/aiXbYIcA6WP3huMCD6/2rth8aSNgNpIjbBIzG3XETvQO/mMTVGOV/qT8xXQo3/z/ADFUuBtSQRp1LIjM6hEGcdaUuGWaffLH1LW92/rf/uNXoKXOZxB+Jsxj4jRNY9foaUeELM7ySrzZalntjFyMie2AxzvtAz7TR/MHcfWg6xC8ur204+pH4VIrDkTvVGwQdht6ZqWkxnfqep9TAGTvXSkAx/n1mgTAeJIShgMT00mCMb5IH+bVhKt1SwbWk82GDsTAXIC8wiOvT3rfvXiAYRjjcQPwYg1leJWgNLN5qiQDN0CRvAPmSDj55HWRRnj1RpGjEqd7fyvyIXrhhyHeRkhhgHVggmAGJI22j60S+TcEso66mXVPQyEeAY9I364p7hkDFWsHUJLNqdiCYIAO/MCAYORFadvhcRcOrGQYgEdZCg/+axYMPiSb7GzLqPDgl3BNfd1hAjDlGo6lzOYBUz9aa4C4SsmZBZYMYKEqYgbSKUu+HJqUacTy5kCYJ5CIjGaEvG6WuLOfMjSig6mGlTplhEsQD0BbfM10ZSppGTFg8SLa5NmpSfD8Rdb/AHYUT95sx7BTn0mnKmUNVyVfce/6Gpd+E+xqPuPf9DUu7H2NAi9hsH/8f5zXPC+IPlgBS0diuJ92n8BVrV9YJLCdIjOZzPzpfw3iZJ1uRE74B26z+HrWajtdS8w3h12UYQZ5jJGO2D+lSg8PxikuGddsSQO+3fpRPMHcVbjMWsd19S1Sq6x6/Q1Nfofp+9WGIXvtc1EIFjGST6zgfKqKLjMCRp0kwYEEbEYeYO/pA7UxqM7HI9Onz9aQvWSraigYGJJRSZ/5ObsfhP7AzTVgRIrjbr/Uv5ilLN199IVCf5pBPUhgsA4wBuT6mmNLalMiNS4j171GXDJ4dskfVFC2l3JLZJxB0jJyIHWe/QVG4sSAASDuY2/ftV/EAnmHUYwD8YHpsSMevpQLa2pyxAG5LrCmJ6PIiKrWRnRnp4Sk5O/f099wzX1IxJPaD+1C88gD7JzjsP1imRw1pQLnmDSdjqwSfWd5rljiVZ9Of/dk7joHJ60eIyv4OF3b+34AsQAWCMTHw6TuOm0Cu4I+Eg/0HH4VsgV2n4jH8JjqtzFvEaTCkmP4T+1cOk7qxHYq0D8K26De4ZWMkttGHYD6KQJ9d6XiMXwcL5f2/BicRZT4ha1EdNBk+kkY70ZLaDIQg+it+1M3fDW2t3Cog/E11jJ66vNGNse+RNUHhj5+1ziM3YBgAyvncwOcY+ZzR4jH8Ljrv/Jn3eGVTqRIbVuFIIwd/wCLP+daQXhG1klLhzuFXIkTu05CRMA8302v9FcSBqZoUwR5kBhJE6r8kZ264yIqlqw8Mum6JwCS+J1AH/7gmBImI29oi3ZdjxrGmk37+gja4NlIcG6SSJEZGwJMyIjoDON840wR/N9GqcPw7B0La4UZJL6cSMzeI2jcGrFCXJ58kx/tNOfZ9MeoqSm0qIZNPCbsFc3EBiJGIbHrkVd4g4I9dB/auGy5xL7D/i7jG/mf58qsVYz8fcYujG3R8nI+hx2fiMq+Dh5v7fgG9wDTFtjJgnScCCZMjuAMd6vImYb20H9q6LbQTzzv/ve3bzNpnHqKL/oniVeDuNRuxkZlfM+najxGSekx/MA9zIhWjM8pwI9qsLoAjOO4P7U8OCWMl/8A3H/+W1XXhVBnmMgiCzEZzsSRR4jF8Hj839vwZbcUJAAJBmTG36maILw9f+k/tWl/p0/hX6CotlQZCgHvAmjxGN6TH8zLa7zDBiDJg429KpxEMMapgx8YHz0xNbTKCIIketL8TwgaIgET0GcR1Bj6UvEYlpILu/t+DFVnZAHVhOoMAs74+9M/jTFq0CyGGZgcMywQCQSMKBGB9KbHAnuDmchP0t/5JovDcFpIJORmISNiNwgPWh5GWRwRjLqjaGSg7CpoHYVKlQLiaB2H0qaB2FSpQBapUqUASpUqUASpUqUASpUqUAcZQRBEg7g7VFUAQBAGwG1SpQB2pUqUASpUqUASpUqUASpUqUASpUqUASpUqUAf/9k="
    alt="The Map of India">

    <br>

<ul>
    <li>Vaibhav</li>
    <li>Shukla</li>
    <li>Ji</li>
</ul><br>
<ol>
    <li>Swami</li>
    <li>Narayan</li>
    <li>Narayan</li>
</ol><br>
<a href="https://www.apple.com/in/store?afid=p238%7Cs8Vs8GkTq-dc_mtid_187079nc38483_pcrid_719882393263_pgrid_112258962467_pntwk_g_pchan__pexid__ptid_kwd-10778630_&cid=aos-IN-kwgo-brand--slid---product-">Visit Apple Website</a>


<!Task 2>
<h2>Registration Form</h2>
<form action="#" method="post">
    
    
    <label for="name">Full Name:</label>
    <input type="text" id="name" name="name" required>
    <br><br>
    <label for="email">Email Address:</label>
    <input type="email" id="email" name="email" required>
    <br><br>
    <label for="dob">Date of Birth:</label>
    <input type="date" id="dob" name="dob" required>
    <br><br>
    <label for="password">Password:</label>
    <input type="password" id="password" name="password" required>
    <br><br>
    <label>Gender:</label>
    <input type="radio" id="male" name="gender" value="male" required> Male
    <input type="radio" id="female" name="gender" value="female" required> Female
    <input type="radio" id="other" name="gender" value="other" required> Other
    <br><br>
    <label>Hobbies:</label>
    <input type="checkbox" id="reading" name="hobbies" value="reading"> Reading
    <input type="checkbox" id="traveling" name="hobbies" value="traveling"> Traveling
    <input type="checkbox" id="sports" name="hobbies" value="sports"> Sports
    <input type="checkbox" id="music" name="hobbies" value="music"> Music
    <br><br>
    <button type="submit">Submit</button>

</form>



</body>
</html>